Musicca/app/src/main/java/com/example/musicca/activities/SongPlaylistActivity.java

```diff
@@ -78,20 +84,134 @@ protected void onCreate(Bundle savedInstanceState) {
         ivPlayPause = findViewById(R.id.ivPlayPause);
         ivNext = findViewById(R.id.ivNext);
 
-        songObjectId = getIntent().getStringExtra(EXTRA_SONGOBJECTID);
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> d632048cbc776aadb255fa324e3b537c1c8edd96
```

Comment:
There seem to be a lot of merge conflicts that are not resolved in this current version.

Reply:
I had not realised there were conflicts, I have managed fixed them though